packages/ai/poetry.lock

```diff
@@ -174,8 +197,8 @@ python-versions = "*"
 version = "0.10.0"
 
 [metadata]
-content-hash = "b94e753039d779e9ae936f7f39fe5a0c9c0ddadff3403553a3e2af17e2264137"
-python-versions = "^3.7.2"
+content-hash = "ba75844de9ca8de3978835a1e28e1ffd0dcbcc74ba6d4375339b33f2602cb6c5"
+python-versions = "^3.7.0"
```

Comment:
This was lowered to `3.7.0` due to the software constraints applied by [Azure Pipelines](https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/linux/Ubuntu1604-README.md)